docs/mobile.md

```diff
@@ -0,0 +1,115 @@
+a corresponding HTLC-timeout. Setting a long CLTV delay at first hop relay offers a _time window_
+for how long the mobile device can be offline. Ideally, the Lightning application will send a
+notification to the user at half of the CLTV delay warning about the requirement to connect back
+to the network.
+
+On the UX-side, as soon the HTLC is sent on the channel, the user balance should be decreased
+accordingly as unlock secret might already be known by counterparty or receiver. In the future,
+the PTLC protocol extension will allow to acknowledge that the payment has been received by 
+expected payee before to decrease balance.
+
+In case of receiving a payment, if it doesn't succeed immediately (e.g waiting on payer interactions
+to receive the preimage), the mobile should be back online to hopefully learn payment preimage
+and thus either claim the received HTLC in cooperation with channel counterparty or worst-case scenario
+to broadcast onchain the commitment transaction and a corresponding HTLC-Success. Requiring a long
+CLTV delay at last hop offers a _time window_ for how long the mobile device can be offline. With
+LDK, this can be easily done by checking `event::PaymentReceived` at reception. Ideally, the 
+Lightning application will send a notification to the user at half of the CLTV delay warning about
+the requirement to connect back to the network.
+
+On the UX-side, the user balance should only be increased when a `revoke_and_ack` message is received
+from the counterparty or the HTLC has been claimed onchain. Otherwise, you can't exclude onchain
+claims races depriving the user from its payment.
+
+## Chain Validation
+
+A Lightning node needs access to the chain to verify channel utxo state. Failure to do so
+might lead to accept payments on a non-existent or already-closed channel, thus provoking a loss of
+funds. As chain validation is a costly processing, a mobile device might not have the bandwidth/CPU/
+storage resources to allocate for.
+
+In the Bitcoin ecosystem, mobile devices have been in practice relying on lightclient protocols
+such as Electrum, BIP37 or BIP157 to access chain data. If the server isn't operated by the mobile
+device user, it should be noted that funds security might be jeopardized by this entity.
+
+LDK offers a ready-to-integrate utility to fetch chain data from a Bitcoin Core node servicing
+BIP157 filters. This utility will produce blocks to the rest of the LDK engine through
+implementing `chain::Listen` interface.
+
+Note, a Lightning client doesn't have to wait syncing with chain tip to start some LN operations.
+Assuming client wallet keys have not leaked, a channel funding sequence should be always valid.
+Further, sending payment is only decreasing user balance. In case of the channel being already
+closed, the user balance committed onchain will be higher, thus at its advantage (and a loss for
+the counterparty...)
```

Comment:
I actually think we can just say "you can run totally normally once you're within the CLTV delay of the channel".

Reply:
You mean CLTV delay minus best-chain tip OR CSV delay ? Effectively I think both holds you're done with headers-sync.